v2/tests/test_parse_xml.py

```diff
@@ -0,0 +1,7 @@
+from robotmk import parse_xml
+
+
+def test_parse_rebot() -> None:
+    with open("v2/tests/rebot.xml", "r", encoding="utf-8") as file:
```

Comment:
Does this also work if I execute only this test?

Reply:
These work no problem:
```
pytest v2/tests/test_parse_xml.py
pytest v2/tests/test_parse_xml.py::test_parse_rebot
```
Or did you mean something else?

Reply:
No, this is what I meant.